FIRMWARE/PocketSDVX/PocketSDVX.ino

```diff
@@ -79,6 +93,15 @@ void lights(uint16_t lightDesc) {
   }
 }
 
+void updateEEPROM(int address, int newValue) {
+  int readState = EEPROM.read(address);
+
+  if (readState == newValue) return;
+
+  EEPROM.update(address, newValue);
+  delay(200);
+}
+
```

Comment:
Refactor updating EEPROM into a function, no need to delay(200) everywhere anymore. Now automatically delay(200) if an update is done